tests/test_edtfdatestring.py

```diff
@@ -45,6 +55,14 @@ def test_load_date():
     pytest.raises(ValidationError, s.load, {"date": "2020-01-01T10:00:00"})
 
 
+def test_load_lvl2date():
+    s = TestSchemaLVL2Date()
+    assert s.load({"date": "2020?"})
+    assert s.load({"date": "2020-01-01"})
+    assert s.load({"date": "2004-06-~01/2004-06-~20"})
+    assert s.load({"date": "2020-01-XX"})
```

Comment:
minor/comment: not really for this PR, but do these work when indexing/dumping them to OpenSearch? And what kind of data do we get out of them? I remember a discussion about adding [Range field type](https://www.elastic.co/docs/reference/elasticsearch/mapping-reference/range) fields in the mapping, but of course it would be greatly out of scope (and probably something we could add as a sub-field in the future).